packages/venia-concept/webpack.config.js

```diff
@@ -45,8 +45,18 @@ module.exports = async env => {
     const storeConfigData = await getStoreConfigData();
     const { availableStores } = await getAvailableStoresConfigData();
 
+    /**
+     * Loop the available stores when there is provided STORE_VIEW_CODE
+     * in the .env file, because should set the store name from the
+     * given store code instead of the default one.
+     */
+    const availableStore = process.env.STORE_VIEW_CODE
+        ? availableStores.find(store => store.code === process.env.STORE_VIEW_CODE)
+        : undefined;
+
     global.MAGENTO_MEDIA_BACKEND_URL = mediaUrl;
     global.LOCALE = storeConfigData.locale.replace('_', '-');
+    global.STORE_NAME = storeConfigData.store_name;
```

Comment:
Do we need the global variable here? `global.MAGENTO_MEDIA_BACKEND_URL` and `global.LOCALE` are used in the `venia-concept/template.html` but I don't think we use `global.STORE_NAME` or `global.AVAILABLE_STORE_VIEWS`, so this just might be leaking detail unnecessarily.

Reply:
Hi @sirugh,
I've added the `global.STORE_NAME` as per the suggestion in #1617.
Please confirm I should delete it.
Thanks!

Reply:
I think the suggestion was indicating that `STORE_NAME` should be used in the `DefinePlugin`, which it already is. Please delete it.